packages/tags/src/Tags.ts

```diff
@@ -104,9 +104,29 @@ export class Tags extends FocusVisiblePolyfillMixin(SpectrumElement) {
         this.rovingTabindexController.clearElementCache();
     }
 
+    private async changeDefaultFocus(event: Event): Promise<void> {
+        if (event.defaultPrevented) {
+            return;
+        }
+        // check if tags are not present
+        if (this.tags && !this.tags.length) {
+            return;
+        }
+        // check if tag item is disabled
+        this.tags.forEach((item) => {
+            if (item.disabled) {
+                return;
+            }
+        });
+        this.rovingTabindexController.changeDefaultItemFocus();
+    }
```

Comment:
Again, this workflow works when the element is removed. Please also test if the element becomes `disabled`.

Reply:
Also this is not needed as of now. Shifted to Mutation Observer